💄 style: Add new model provider Novita AI

jasonhp opened the pull request:
#### 💻 变更类型 | Change Type

<!-- For change type, change [ ] to [x]. -->

- [x] ✨ feat
- [x] 🐛 fix
- [ ] ♻️ refactor
- [x] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change

1. Add new model provider: Novita AI
2. FIx test case: OPENROUTER_MODEL_LIST, and its snapshot.

<!-- Thank you for your Pull Request. Please provide a description above. -->

#### 📝 补充信息 | Additional Information

<!-- Add any other context about the Pull Request here. -->

jasonhp committed:
feat: add Novita AI as new model provider.

jasonhp committed:
feat: add Novita AI as new model provider.

jasonhp committed:
Merge remote-tracking branch 'upstream/main' into provider_novita

jasonhp committed:
feat: update Novita AI introduction in README.md; fix OPENROUTER_MODEL_LIST test case and snapshot

arvinxx committed:
Merge branch 'main' into provider_novita

arvinxx commented on src/app/(main)/settings/llm/ProviderList/providers.tsx:
```diff
@@ -119,6 +121,11 @@ export const useProviderList = (): ProviderItem[] => {
         docUrl: urlJoin(BASE_DOC_URL, 'groq'),
         title: <GroqBrand />,
       },
+      {
```
请放在 OpenRouter 后面，目前 OpenRouter 的用户还是比较多的，它需要靠前

jasonhp replied:
已调整

arvinxx commented on src/config/modelProviders/baichuan.ts:
这些为啥被格式化了？

jasonhp replied:
已恢复

arvinxx commented on src/libs/agent-runtime/baichuan/index.ts:
这里也是，为什么被格式化了

jasonhp replied:
类似格式化均已恢复

cy948 committed:
:recycle: refactor: Support disable clientFetch by default (#3133)

* ♻️ refactor: Support disable clientFetch by modelConfig (#3108)

* :white_check_mark: test: - clientfetch for qwen

* ♻️ refactor: Remove `defaultShowBrowserRequest` in qwen modelConfig

semantic-release-bot committed:
:bookmark: chore(release): v1.3.4 [skip ci]

### [Version&nbsp;1.3.4](https://github.com/lobehub/lobe-chat/compare/v1.3.3...v1.3.4)
<sup>Released on **2024-07-09**</sup>

#### ♻ Code Refactoring

- **misc**: Support disable clientFetch by default.

<br/>

<details>
<summary><kbd>Improvements and Fixes</kbd></summary>

#### Code refactoring

* **misc**: Support disable clientFetch by default, closes [#3133](https://github.com/lobehub/lobe-chat/issues/3133) [#3108](https://github.com/lobehub/lobe-chat/issues/3108) ([4415652](https://github.com/lobehub/lobe-chat/commit/4415652))

</details>

<div align="right">

[![](https://img.shields.io/badge/-BACK_TO_TOP-151515?style=flat-square)](#readme-top)

</div>

arvinxx committed:
🐛 fix: fix assistant meta change race issue (#3184)

semantic-release-bot committed:
:bookmark: chore(release): v1.3.5 [skip ci]

### [Version&nbsp;1.3.5](https://github.com/lobehub/lobe-chat/compare/v1.3.4...v1.3.5)
<sup>Released on **2024-07-10**</sup>

#### 🐛 Bug Fixes

- **misc**: Fix assistant meta change race issue.

<br/>

<details>
<summary><kbd>Improvements and Fixes</kbd></summary>

#### What's fixed

* **misc**: Fix assistant meta change race issue, closes [#3184](https://github.com/lobehub/lobe-chat/issues/3184) ([6335be4](https://github.com/lobehub/lobe-chat/commit/6335be4))

</details>

<div align="right">

[![](https://img.shields.io/badge/-BACK_TO_TOP-151515?style=flat-square)](#readme-top)

</div>

jasonhp committed:
fix: fix provider order; fix auto formatting;

jasonhp committed:
fix: fix auto formatting

arvinxx reviewed (commented):
还需要补充一篇配置文档，参考：https://lobehub.com/zh/docs/usage/providers/qwen

arvinxx committed:
♻️ refactor: improve agent runtime code (#3199)

* ♻️ refactor: improve agent runtime code

* ♻️ refactor: clean code

semantic-release-bot committed:
:bookmark: chore(release): v1.3.6 [skip ci]

### [Version&nbsp;1.3.6](https://github.com/lobehub/lobe-chat/compare/v1.3.5...v1.3.6)
<sup>Released on **2024-07-11**</sup>

#### ♻ Code Refactoring

- **misc**: Improve agent runtime code.

<br/>

<details>
<summary><kbd>Improvements and Fixes</kbd></summary>

#### Code refactoring

* **misc**: Improve agent runtime code, closes [#3199](https://github.com/lobehub/lobe-chat/issues/3199) ([9f211e2](https://github.com/lobehub/lobe-chat/commit/9f211e2))

</details>

<div align="right">

[![](https://img.shields.io/badge/-BACK_TO_TOP-151515?style=flat-square)](#readme-top)

</div>

lobehubbot committed:
📝 docs(bot): Auto sync agents & plugin to readme

RubuJam committed:
🔨 chore: fix netlify deploy error (#3201)

jasonhp committed:
fix: fix novita README introduction

jasonhp committed:
Merge branch 'main' into provider_novita

arvinxx commented on src/app/(main)/settings/llm/ProviderList/providers.tsx:
```diff
@@ -119,11 +121,21 @@ export const useProviderList = (): ProviderItem[] => {
+        ...NovitaProviderCard,
+        docUrl: urlJoin(BASE_DOC_URL, 'novita'),
+        title: <Novita.Combine size={20} type={'color'} />,
+      },
```
怎么重复添加了？

jasonhp replied:
已修复

arvinxx commented on src/components/ModelProviderIcon/index.tsx:
```diff
@@ -116,6 +117,10 @@ const ModelProviderIcon = memo<ModelProviderIconProps>(({ provider }) => {
       return <Stepfun size={20} />;
     }
 
+    case ModelProvider.Novita: {
```
这里也是

jasonhp replied:
所有重复添加部分都已修复

arvinxx reviewed (changes requested):
是不是合并有问题，出现了很多重复添加的情况

jasonhp committed:
fix: fix repeated code; add docs for novita

jasonhp committed:
Merge branch 'main' into provider_novita

arvinxx committed:
Update novita.mdx

arvinxx committed:
Update novita.zh-CN.mdx

arvinxx committed:
Merge branch 'main' into provider_novita

jasonhp committed:
fix: add disableBrowserRequest config

arvinxx reviewed (approved):
LGTM now, Thanks for your great work!

xenstar commented:
@jasonhp Seems the get model list button doesn't work. It can't get the latest Llama 3.1 models. 

![image](https://github.com/user-attachments/assets/aa7c3a23-43d0-415b-bd79-f44a9c935543)
